static/src/javascripts/projects/common/modules/experiments/tests/platform-dont-upgrade-mobile-rich-links.js

```diff
@@ -0,0 +1,32 @@
+define([], function () {
+    return function () {
+        this.id = 'DontUpgradeMobileRichLinks';
+        this.start = '2016-08-18';
+        this.expiry = '2016-09-14';
+        this.author = 'Gareth Trufitt';
+        this.description = 'Test whether the loyalty of users decreases with non-enhanced rich links';
+        this.audience = 0.5;
+        this.audienceOffset = 0;
+        this.successMeasure = 'Loyalty of users';
```

Comment:
It would be more accurate to say we're testing _engagement_, notably that there is no drop in overall CTR on the page, or article visits per browser per day.